src/main/java/org/springframework/data/repository/reactive/RxJava1CrudRepository.java

```diff
@@ -73,7 +73,7 @@
 	Observable<T> findOne(ID id);
 
 	/**
-	 * Retrieves an entity by its id supplied by a {@link Single}.
+	 * Returns whether an entity with the given id, supplied by a {@link Single}, exists.
```

Comment:
oopsie, that's the wrong method you changed the javadoc on, for `RxJava1CrudRepository` :p